pwndbg/commands/context.py

```diff
@@ -155,8 +157,16 @@ def context_source():
         if not source or source.startswith('1\t'):
             return []
 
+        # highlight the current code line
+        source_lines = source.splitlines()
+        if pwndbg.config.highlight_source:
+            for i in range(len(source_lines)):
```

Comment:
This should probably use symbol-table lookups to find the current line.

There's an example here:
https://sourceware.org/gdb/onlinedocs/gdb/Line-Tables-In-Python.html

```py
     symtab = gdb.selected_frame().find_sal().symtab
     if symtab is not None:
         for line in symtab.linetable():
             print "Line: "+str(line.line)+" Address: "+hex(line.pc)
```